monkey/infection_monkey/telemetry/aws_instance_telem.py

```diff
@@ -0,0 +1,20 @@
+from common.common_consts.telem_categories import TelemCategoryEnum
+from infection_monkey.telemetry.base_telem import BaseTelem
+
+
+class AwsInstanceTelemetry(BaseTelem):
+    def __init__(self, aws_instance_info):
+        """
+        Default AWS instance telemetry constructor
+        :param aws_instance_info: Aws Instance info
+        """
+        super(AwsInstanceTelemetry, self).__init__()
```

Comment:
What's the point of this call?

Reply:
This can be the [answer](https://stackoverflow.com/questions/576169/understanding-python-super-with-init-methods
). We are using it in all of the telemetries. Although based on the answer we can use `super().__init()` and it is the same as above but nicer.

Reply:
I think he means that the `__init__()` function isn't defined in BaseTelem. Calling it doesn't do anything.